scripts/policy/frameworks/cluster/controller/util.zeek

```diff
@@ -0,0 +1,17 @@
+module ClusterController::Util;
+
+export {
+	global set_to_vector: function(ss: set[string]): vector of string;
```

Comment:
This should be documented. Also - should this one perhaps be renamed something like string_set_to_vector and moved to `base/utils/strings.zeek`?